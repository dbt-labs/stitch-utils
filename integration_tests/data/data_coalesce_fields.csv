"id","numeral","numeral__st","numeral__bo","version__string","version__double","""bad column""","""bad column with type__string""","""bad column with type__int"""
1,1,,,pi,,testy,zero,
2,,one,,,3.14,testoh,,0
3,,,TRUE,,,testah,nil,
