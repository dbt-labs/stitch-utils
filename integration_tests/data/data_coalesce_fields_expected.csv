"id","numeral","version","""bad column""","""bad column with type"""
1,"1","pi","testy","zero"
2,"one","3.14","testoh","0"
3,"true",,"testah","nil"